docs/general/ddl/replication.rst

```diff
@@ -1,73 +1,85 @@
-contains a ``range``.
-
-A range is a definition of minimum number of replicas to maximum number of
-replicas depending on the number of nodes in the cluster. The table below shows
-some examples.
-
-===== =========================================================================
-Range Explanation
-===== =========================================================================
-0-1   Will create no replicas if only one node. This will result in a yellow
-      cluster health.
-
-      One replica if more than one node.
------ -------------------------------------------------------------------------
-2-4   Table requires at least two replicas to be fully replicated.
-
-      Will create up to four replicas if nodes are added.
-
-      If you have less than three nodes, one or more replica shards will be
-      located on the same node as the primary shard. This will result in a
-      yellow cluster health.
------ -------------------------------------------------------------------------
-0-all Will expand the number of replicas to the available number of nodes.
-===== =========================================================================
-
-For details of the range syntax refer to :ref:`the CREATE TABLE
-documentation <sql-create-table-number-of-replicas>`.
-
-.. NOTE::
-
-    The number of replicas can be changed at any time.
+As well as configuring a single fixed number of pre-replica shards, you can use
+a string to configure a range by specifying a minimum and a maximum (dependent
+on the number of nodes in the cluster).
+
+Here are some examples of replica ranges:
+
+========= =====================================================================
+Range     Explanation
+========= =====================================================================
+``0-1``   If you only have one node, CrateDB will not create any replicas.
+          Having no replicas will result in a *yellow* cluster health. If you
+          have more than one node, CreateDB will create one replica per shard.
+--------- ---------------------------------------------------------------------
+``2-4``   Each table will require at least two replicas for CrateDB to consider
+          it fully replicated (i.e., a *green status*).
+
+          If the cluster has four nodes, CrateDB will create four replicas,
+          with each replica located on a different node from its respective
+          primary to improve :ref:`resiliency <concept-resiliency>` (e.g., due
+          to node failure).
+
+          If a cluster has three or fewer nodes, CreateDB will have to locate one
+          or more  replica shards will be on the same node as the respective
+          primary shard. If this happens, your cluster will have a *yellow*
+          cluster health.
+--------- ---------------------------------------------------------------------
+``0-all`` CrateDB will create as many replicas as you have nodes available.
```

Comment:
I want to avoid the inline literals if possible. how about this:

> CrateDB will create one replica shard for every node that is available in addition to the node that holds the primary shard.